Sources/GDSCommon/UI/UIKit/SharedViews/Patterns/GDSInformation/GDSInformationViewController.swift

```diff
@@ -0,0 +1,107 @@
+import UIKit
+
+/// View controller for `GDSInformation` screen
+///     - `informationImage` (type: `String`)
+///     - `titleLabel` (type: `UILabel`)
+///     - `bodyLabel` (type: `UILabel`)
+///     - `footnoteLabel`  (type: `UILabel`)
+///     - `primaryButton`  (type: ``RoundedButton`` inherits from ``SecondaryButton``)
+///     - `secondaryButton`  (type: ``SecondaryButton`` inherits from ``UIButton``)
+public final class GDSInformationViewController: BaseViewController, TitledViewController {
+    public override var nibName: String? { "GDSInformation" }
+    
+    public private(set) var viewModel: GDSInformationViewModel
+    
+    public init(viewModel: GDSInformationViewModel) {
+        self.viewModel = viewModel
+        super.init(viewModel: viewModel as? BaseViewModel, nibName: "GDSInformation", bundle: .module)
+    }
+    
+    @available(*, unavailable, renamed: "init(coordinator:)")
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    @IBOutlet private var informationImage: UIImageView! {
+        didSet {
+            let font = UIFont(style: .largeTitle, weight: viewModel.imageWeight ?? .semibold)
+            let configuration = UIImage.SymbolConfiguration(font: font, scale: .large)
+            informationImage.image = UIImage(systemName: viewModel.image, withConfiguration: configuration)
+            informationImage.tintColor = viewModel.imageColour ?? .gdsPrimary
+            informationImage.accessibilityIdentifier = "information-image"
+            
+            NSLayoutConstraint.activate([
+                informationImage.heightAnchor.constraint(greaterThanOrEqualToConstant: viewModel.imageHeightConstraint ?? 55)
```

Comment:
Did you look at figuring out the calculation for transforming the `viewModel.imageHeightConstraint` so that devs inputting a height for the image get the height they expect?

Reply:
Currently compensating the padding by adding 11, and scaling set to large